builtin/mainmenu/serverlistmgr.lua

```diff
@@ -90,8 +90,11 @@ function serverlistmgr.sync()
 end
 
 --------------------------------------------------------------------------------
-local function get_favorites_path()
+local function get_favorites_path(folder)
 	local base = core.get_user_path() .. DIR_DELIM .. "client" .. DIR_DELIM .. "serverlist" .. DIR_DELIM
+	if folder then
```

Comment:
```suggestion
local function get_favorites_path(get_path_only)
	local base = core.get_user_path() .. DIR_DELIM .. "client" .. DIR_DELIM .. "serverlist" .. DIR_DELIM
	if get_path_only then
```

Reply:
The favourites path is the path to the file